### ISSUE TITLE
[zstd] Expose the library version

### ISSUE THREAD

**Issue by Morten242** [microsoft/vcpkg#13132]
We want to limit which version of zstd is used to 1.3 or above and while 1.4 is provided by vcpkg it doesn't expose the version information though the zstd-config.cmake or similar, so it is always rejected.

I don't know if this a common thing for the *-config.cmake files, but I would like to have the version info in there so they can be used in version-specific find_package calls.

I also don't know what it would take to do so, but I would imagine the cmake could generate this and I'm not sure why it isn't already? In any case for now I'm working around it by removing the version from the find_package calls locally.

**Comment by LilyWangL** [comment]
Thanks for posting this issue. Do you mean want use `find_package(zstd 1.4 CONFIG REQUIRED)` to find zstd or want install special version zstd?

**Comment by Morten242** [comment]
Yes, this is about specifying the version in find_package. Currently it will just say something to the effect of 'couldn't find zstd (requires version "1.4", found version "")'

**Comment by LilyWangL** [comment]
Related issue #12848

**Comment by LilyWangL** [comment]
I think this also need upstream export project-config-version.cmake.
